### DISCUSSION TITLE
unable to get instance by host: instanceHost netbird.XXX.com, publicHost : ID=QUERY-1kIjX Message=Errors.IAM.NotFound"

### DISCUSSION THREAD

**Discussion by zz940522** [zitadel/zitadel#8873] [Q&A] [answered]
Hello there, I want to depoy **netbird** with **zitadel** .
I already deploy zitadel successfully,and i can login into ui.
Then i make some config for netbird
And then ,i deployed **netbird** ,then i use netbird.XXX.com 
Finally, show errors "unable to get instance by host: instanceHost netbird.XXX.com, publicHost : ID=QUERY-1kIjX Message=Errors.IAM.NotFound"

BTW, zitadel external domain is a subdomain 'zitadel.XXX.com' ,XXX.com is the domain.

How can i fix this ?

**Comment by fforootd** [comment]
Can you share the Zitadel config and how you deployed Zitadel?

**Reply by zz940522** [reply, marked as answer]
Finally , i tried docker compose down and then docker compose up -d ,work well.
Thank you !!!
